### ISSUE TITLE
File Dialogs always open at the bottom of my screen

### ISSUE THREAD

**Issue by tiptronic** [bramblex/niva#19]
In NivaDevTools file-dialogs (e.g. when clicking 'Open Project' or 'New Project') always open at the lower left of my monitor (2 displays connected, if that is of any worth).

MacOS Ventura 13.3.1

**Comment by bramblex** [comment]
Yes, this is a known bug.  Niva uses (rfd)[https://github.com/PolyMeilex/rfd] to open dialogs, and this is a bug in `rfd`. I will try to find a solution to this bug in the near future. 😢
